src/Umbraco.Web.UI.Client/src/views/propertyeditors/blockgrid/prevalue/umb-block-grid-area-editor.html

```diff
@@ -0,0 +1,33 @@
+<div class="umb-block-grid-area-editor">
+
+    <umb-load-indicator ng-if="vm.loading"></umb-load-indicator>
+
+    <div ng-show="vm.loading !== true" class="umb-block-grid-area-editor__grid-wrapper" style="--umb-block-grid--block-grid-columns: {{vm.block.areaGridColumns || vm.rootLayoutColumns}}">
+
+        <umb-block-grid-configuration-area-entry ng-repeat="area in vm.model track by area.key"
+            class="umb-block-grid-area-editor__area"
+            ng-class="{'--isOpen':vm.openArea === area}"
+            area="area"
+            style="
+            --umb-block-grid--area-column-span: {{area.columnSpan}};
+            --umb-block-grid--area-row-span: {{area.rowSpan}};
+            "
+            ng-click="vm.editArea(area)"
+            on-edit="vm.editArea(area)"
+            on-delete="vm.requestDeleteArea(area)"
+        >
+        </umb-block-grid-configuration-area-entry>
+
+        <button
+            type="button"
+            class="btn-reset umb-block-grid-area-editor__create-button umb-outline"
+            ng-click="vm.onNewAreaClick()">
+        <localize key="blockEditor_TODO">Add area</localize>
```

Comment:
I guess this should be `blockEditor_addArea` :)

Reply:
Oh yes, all localizations are still to be decided/done.. :-)

Reply:
Should be solved now